.csslintrc

```diff
@@ -20,6 +20,7 @@
     "qualified-headings": false,
     "regex-selectors": false,
     "selector-max-approaching": false,
+	"selector-max": false,
```

Comment:
Spacing, but this also means that the compiled CSS has exceeded 32K selectors I believe